test/functional/wallet_address_types.py

```diff
@@ -348,9 +348,9 @@ def run_test(self):
         if self.options.descriptors:
```

Comment:
It seems now the change type behavior for both type of wallets is same. So this can be removed?

Reply:
I think the whole test needs to be re-written from scratch, but I want to keep the changes minimal here.